src/chains/ethereum/ethereum/src/forking/trie.ts

```diff
@@ -291,7 +291,7 @@ export class ForkTrie extends GanacheTrie {
    */
   copy(includeCheckpoints: boolean = true) {
     const db = this.db.copy() as CheckpointDB;
-    const secureTrie = new ForkTrie(db._leveldb, this.root, this.blockchain);
+    const secureTrie = new ForkTrie(db._leveldb as LevelUp, this.root, this.blockchain);
```

Comment:
`merkle-patricia-tree` uses a different version of the `CheckpointDB` types so we have to cast it to our own.